pkg/flowkit/util/utilities.go

```diff
@@ -187,3 +190,57 @@ func AddToGitIgnore(filename string, loader ReaderWriter) error {
 		filePermissions,
 	)
 }
+
+type jsonConfig struct {
+	Metrics bool `json:"metrics"`
+}
+
+// AddToConfig adds a new line to user's config dir stating the user's preference for command tracking
+func AddToConfig(loader ReaderWriter, metricsEnabled bool) error {
+	homeConfigDir, err := os.UserConfigDir()
+	if err != nil {
+		return err
+	}
+	cliDir := filepath.Join(homeConfigDir, FLOW_CLI)
```

Comment:
we use filepath.join because it will automatically append \ for windows and / for other os.